client/src/assets/Images/Spielplatzfinder-Dateien/main.js

```diff
@@ -0,0 +1,2485 @@
+(this["webpackJsonpclient"] = this["webpackJsonpclient"] || []).push([["main"],{
```

Comment:
hm 🤨

Reply:
i didnt create that

Reply:
This looks like you've built your app, and moved the build folder contents inside here or something like this